contracts/test/hashes.ts

```diff
@@ -41,4 +37,18 @@ describe('Hashes', () => {
   it('SHA384', async () => {
     await testHashes('SHA384', contract.testSHA384.bind(contract));
   });
+
+  it('HMAC SHA512-256', async () => {
+    for (let i = 0; i < 1024; i = i + (1 + i / 5)) {
```

Comment:
Is there a specific reason to use `/ 5`?

Reply:
Because it's a nice sequence that tests boundaries key length and varying message sizes, without testing too many of them. I'm unaware of any standard test vectors for HMAC SHA512-256.